# This is real ephemeris data
# You can find it for example here : https://celestrak.org/NORAD/elements/

RE: 6378000

muE: 3.986004418e+14

sqrt_sa: 
  - 5153.653320
  - 5153.606445
  - 5153.542969
  - 5153.562012
  - 5153.740723
  - 5153.577637
  - 5153.717285
  - 5153.704102
  - 5153.624512
  - 5153.617188
  - 5153.59912
  - 5153.634766
  - 5153.541016
  - 5153.624512
  - 5153.543457
  - 5153.713867
  - 5153.660645
  - 5153.585938
  - 5153.533691
  - 5153.751465
  - 5153.631348
  - 5153.538086
  - 5153.669922
  - 5153.639160
  - 5153.549316
  - 5153.604980
  - 5153.591309
  - 5153.683105
  - 5153.708008
  - 5153.531250
  - 5153.600586

mu_anomalie:
  - -0.1208933778e+001
  - -0.7379312701e+000
  - -0.2476050596e+001
  - 0.1729284331e+001
  - -0.5955965034e+000
  - -0.1863089325e+001
  - 0.1717615066e+001
  - 0.1041308928e+001
  - 0.3003120927e+001
  - -0.1772399827e+001
  - -0.2456528294e+001
  - -0.8542115828e+000
  - 0.4893275126e+000
  - 0.1755864967e+000
  - 0.1876034910e+001
  - 0.8991835103e+000
  - 0.2479146271e+001
  - -0.2239168159e+001
  - 0.5016783799e-001
  - -0.2845982272e+001
  - 0.4089099929e+000
  - -0.3331816757e+000
  - -0.8946785652e+000
  - -0.2992513015e+001
  - 0.2928989882e+001
  - 0.9610869005e+000
  - 0.1794198757e+001
  - 0.2993612193e+001
  - -0.1855898041e+001
  - -0.2008106311e+001
  - 0.1461498698e+001

arg:
  - 0.563527841
  - -2.074941210
  - 0.718299986
  - 0.536422146
  - -1.700786593
  - -2.620313325
  - -1.011684673
  - 1.869199782
  - -2.911753571
  - 1.589303341
  - 0.783939084
  - 1.923932861
  - -1.944528126
  - 0.517680691
  - 0.382938305
  - -1.917666234
  - -1.854486524
  - 0.879175098
  - 1.466748537
  - -1.748448979
  - -1.854394021
  - -2.525418487
  - 0.394514692
  - 0.771921528
  - -0.184188174
  - 0.328577112
  - -1.607607372
  - 0.029954945
  - -3.122813373
  - -0.369073306
  - -2.439273631

orbinc:
  - 0.9651879023
  - 0.9440357453
  - 0.9595553166
  - 0.9465644168
  - 0.9649961547
  - 0.9644988093
  - 0.9649302414
  - 0.9547136897
  - 0.9597590484
  - 0.8981062034
  - 0.9901450506
  - 0.9709223539
  - 0.9634382054
  - 0.9304156737
  - 0.9906543802
  - 0.9780709440
  - 0.9252444807
  - 0.9752067143
  - 0.9265148085
  - 0.9373066030
  - 0.9232011703
  - 0.9456176630
  - 0.9475830759
  - 0.9782207469
  - 0.9601005988
  - 0.9730315775
  - 0.9893960366
  - 0.9790955953
  - 0.9502675423
  - 0.9692924993
  - 0.9592676952

RAANangle:
  - 0.1489908052e+001
  - 0.1441165588e+001
  - 0.2532003819e+001
  - 0.2518117098e+001
  - 0.1481505987e+001
  - -0.1634831037e+001
  - 0.4283431626e+000
  - -0.2713451725e+001
  - 0.2528123551e+001
  - 0.1102367430e+001
  - -0.5551591071e+000
  - -0.2590521291e+001
  - -0.2628516527e+001
  - -0.2769660736e+001
  - -0.5363502406e+000
  - 0.4829410443e+000
  - 0.2493819082e+001
  - 0.5313011368e+000
  - 0.2442215384e+001
  - 0.1452076850e+001
  - 0.2494154640e+001
  - -0.2709604040e+001
  - -0.1684579427e+001
  - -0.6078234089e+000
  - -0.6194110309e+000
  - 0.4322024576e+000
  - -0.5315595467e+000
  - 0.4928152966e+000
  - -0.1592779889e+001
  - -0.1624817468e+001
  - -0.2711327521e+001

eccen:
  - 0.5968570709e-002
  - 0.1603031158e-001
  - 0.2923011780e-003
  - 0.4848480225e-002
  - 0.2102851868e-003
  - 0.1005315781e-001
  - 0.1864910126e-002
  - 0.6208419800e-003
  - 0.1812458038e-002
  - 0.1681089401e-001
  - 0.6042003632e-002
  - 0.4010200500e-002
  - 0.8724212646e-002
  - 0.8650302887e-002
  - 0.8711338043e-002
  - 0.1133203506e-001
  - 0.1726150513e-001
  - 0.1039505005e-001
  - 0.4734039307e-002
  - 0.2335453033e-001
  - 0.7355213165e-002
  - 0.1113939285e-001
  - 0.4827976227e-002
  - 0.5622386932e-002
  - 0.1107215881e-002
  - 0.3872394562e-002
  - 0.2001571655e-001
  - 0.5064010620e-003
  - 0.2400398254e-002
  - 0.8392333984e-002
  - 0.6222724915e-003

linearization_point: 
# choose this to be around your region of interested. It doesnt need to be precise, 
# just close enough so the curvature of the earth doesnt make the linearization invalide
- 47.36667
- 8.55
- 0 
 
